src/dms/run.sh

```diff
@@ -34,5 +34,17 @@ else
   echo "Skipping Data Management Service schema installation."
 fi
 
-dotnet EdFi.DataManagementService.Frontend.AspNetCore.dll
+if [ "$USE_API_SCHEMA_PATH" = true ]; then
+    echo "Using Api Schema Path."
+
+    CORE_PACKAGE=EdFi.DataStandard52.ApiSchema.Core
+    TPDM_PACKAGE=EdFi.DataStandard52.ApiSchema.TPDM
+
+    echo "Downloading Package ${CORE_PACKAGE}..."
+    dotnet /app/ApiSchemaDownloader/EdFi.DataManagementService.ApiSchemaDownloader.dll -p ${CORE_PACKAGE} -d ${API_SCHEMA_PATH}
 
+    echo "Downloading Package ${TPDM_PACKAGE}..."
+    dotnet /app/ApiSchemaDownloader/EdFi.DataManagementService.ApiSchemaDownloader.dll -p ${TPDM_PACKAGE} -d ${API_SCHEMA_PATH}
```

Comment:
you have to package version also otherwise sometime we will have code changes only in MetaED and not DMS . That time it will break in DMS codebase

Reply:
you can dynamically take package version from this file \Data-Management-Service\src\Directory.Packages.props